packages/components/src/internal/components/search/SearchResultCard.tsx

```diff
@@ -57,8 +57,9 @@ export class SearchResultCard extends React.Component<SearchResultProps, any> {
         }
 
         const { iconSrc, altText } = cardData;
+        const iconDir = cardData.iconDir ?? '_images';
```

Comment:
This actually isn't necessary to set as the default is already "_images". I would just pass along `iconDir`.